database-commons/src/main/java/io/cdap/plugin/db/DBRecord.java

```diff
@@ -261,7 +261,7 @@ private void writeToDataOut(DataOutput out, Schema.Field field) throws IOExcepti
     }
   }
 
-  private void writeToDB(PreparedStatement stmt, Schema.Field field, int fieldIndex) throws SQLException {
+  protected void writeToDB(PreparedStatement stmt, Schema.Field field, int fieldIndex) throws SQLException {
     String fieldName = field.getName();
```

Comment:
Allows customizing the way a field will be set on the `PreparedStatement` instance. For example, we can not set  `null` values of `GEOGRAPHY` and `GEOMETRY` fields  using regular `stmt.setNull(sqlIndex, GEOMETRY_TYPE)` since it leads to 
```
com.microsoft.sqlserver.jdbc.SQLServerException: The conversion from OBJECT to GEOMETRY is unsupported.
```